packages/app-store/wipemycalother/DESCRIPTION.md

```diff
@@ -1,3 +1,8 @@
+---
+items:
+  - /api/app-store/wipemycalother/1.jpg
```

Comment:
```suggestion
  - 1.jpg
```
@PeerRich Btw you can do just 1.jpg now.

Reply:
oh nice!

Reply:
ah shit, wasnt merged. does both work?